### ISSUE TITLE
package as pip

### ISSUE THREAD

**Issue by micheelengronne** [smarkets/marge-bot#209]
As marge-bot is written in python, it could be interesting to package it as a pip package.

It will greatly improve the portability of the project and the integration in other solutions.

**Comment by aschmolck** [comment]
Yes, I'd like to do that, but it's not particularly high priority.
